packages/multimedia/SDL2/patches/SDL2-2.0.12.001.patch

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
I thought I commented on this but I guess not. Why is this patch needed? Likely we would need SDL2 to support EGL as we enable OpenGL/ES support in SDL2.

Reply:
In a further commit I had updated to SDL2 and was able to remove the patch.